### ISSUE TITLE
Can't encrypt with Fedora Silverblue 41 + latest flathub version

### ISSUE THREAD

**Issue by Froggy232** [konstantintutsch/Lock#21]
### Describe the bug …

Hi,
I can see the keys without any problems, but when I try to encrypt a file or text it won't process.
The thing is it works with GPG-Frontend, available on flathub so I think the problem don't came from the key.
Thanks for your soft anyway, it's great to be able to encrypt and decrypt through gpg with gtk.
Thanks you!

### Which version of Lock are you using?

1.0.2

### Source

Flathub

### Relevant log output

```shell
** (com.konstantintutsch.Lock:2): WARNING **: 19:33:05.620: Failed to encrypt GPGME data from memory: Clef publique inutilisable
```

**Comment by konstantintutsch** [comment]
My very limited understanding of the French language is telling me that the public key you want to encrypt for is indeed the problem.

I'm guessing that the public key does not contain an encryption subkey. You might want to ask the person from whom you got the key to reexport their keypair's public encryption key :)

**Comment by konstantintutsch** [comment]
Feel free to reopen this issue if that does not fix the problem!

**Comment by konstantintutsch** [comment]
This could have also been imprecise error reporting from GPGME and issues with using untrusted keys for encryption processes.

If you did not trust the key, this should be fixed in b4dce5154b028c1db05497d0ad098590fcc2367b
